<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deleteCustomer</class>
 <widget class="QDialog" name="deleteCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Member</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_address1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>address1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_address2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>address2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rating">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>address2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_key">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>address2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Member</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
